com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Input/Textures/SampleTriplanarNode.cs

```diff
@@ -0,0 +1,193 @@
+using System.Collections.Generic;
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+    internal class SampleTriplanarNode : IStandardNode
+    {
+        public static string Name = "SampleTriplanar";
+        public static int Version = 1;
+
+        public static NodeDescriptor NodeDescriptor => new(
+            Version,
+            Name,
+            new FunctionDescriptor[] {
+                new(
+                    1,
+                    "SampleTriplanarDefault",
+@"  Node_UV = Position * Tile;
+    Node_Blend = SafePositivePow_float(Normal, min(Blend, floor(log2(Min_float())/log2(1/sqrt(3)))) );
+    Node_Blend /= dot(Node_Blend, 1.0);
+    Node_X = SAMPLE_TEXTURE2D(UnityBuildTexture2DStructNoScale(Texture).tex, Sampler.samplerstate, Node_UV.zy);
```

Comment:
Looks like the `UnityBuildTexture2DStructNoScale()` will be automatically added when compiling the shader, I think `Texture.tex` should be good here

Reply:
Fixed - thank you!